designer/server/src/main/scala/pl/touk/nussknacker/ui/api/UsageReportsSettings.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.ui.api
+
+import io.circe.generic.JsonCodec
+
+import scala.util.Random
+import pl.touk.nussknacker.engine.version.BuildInfo
+
+import java.net.URLEncoder
+import java.util.Base64
+
+private object FingerprintUtils {
+  lazy val random = Random.alphanumeric.take(10).mkString
+}
+
+case class UsageReportsConfig(enabled: Boolean, fingerprint: String = FingerprintUtils.random)
+
+object UsageReportsSettings {
+  def apply(config: UsageReportsConfig): UsageReportsSettings = {
+    val version = Base64.getEncoder.encodeToString(BuildInfo.version.getBytes())
```

Comment:
I'd rather like to have here "sanitized" version but still user friendly. How would you like to present it/use in search on elastic? Let's say that you want all usages with version starting from 1.6.0.
By sanitization I mean e.g. all characters, numbers, .-_ - other things replaced by let's say "_"